apollo-gradle-plugin/src/main/kotlin/com/apollographql/apollo3/gradle/api/Service.kt

```diff
@@ -435,6 +435,13 @@ interface Service {
    */
   val codegenModels: Property<String>
 
+  /**
+   * When to add __typename. One of "ifFragments", "ifAbstract" or "ifPolymorphic"
+   *
+   * Default value: "ifAbstract"
```

Comment:
```suggestion
   * Default value: "ifFragments"
```

Reply:
Maybe we can copy the nice explanation of each value you put in the description of this PR here?